docs/core/prerequisites.md

```diff
@@ -0,0 +1,463 @@
+---
+title: Prerequisites for .NET Core on Windows, Linux, MacOS
+description: Supported Windows, Linux, MacOS versions and .NET Core dependencies to develop, deploy, and run .NET Core applications.
+author: leecow
+ms.author: leecow
+ms.date: 10/30/2019
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Prerequisites for .NET Core
+
+This article provides information on getting started with .NET Core. .NET Core can be installed on Windows, Linux, and macOS. You can code in your favorite text editor and produce cross-platform libraries and applications.
+
+.NET Core is made up of two components, the runtime and the SDK. The **runtime** is required to run a .NET Core application that doesn't include the runtime. The **SDK** is required to build .NET Core applications.
+
+There are different kinds of ways to develop .NET Core apps:
+
+::: zone pivot="os-windows"
+
+- [Command-line with your favorite editor](tutorials/using-with-xplat-cli.md)
+- [Visual Studio](https://www.visualstudio.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2019)
+- [Visual Studio Code](https://code.visualstudio.com/)
+
+::: zone-end
+
+::: zone pivot="os-linux"
+
+- [Command-line with your favorite editor](tutorials/using-with-xplat-cli.md)
+- [Visual Studio Code](https://code.visualstudio.com/)
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+- [Command-line with your favorite editor](tutorials/using-with-xplat-cli.md)
+- [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link).
+- [Visual Studio Code](https://code.visualstudio.com/)
+
+::: zone-end
+
+You can download a standalone installer for each OS from the following links:
+
+- [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+- [.NET Core 2.2 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.2).
+- [.NET Core 2.1 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.1).
+
+It's important to understand the [.NET Core support policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) when choosing which version of .NET Core to develop apps for.
+
+## Is .NET Core installed
+
+It's possible that the .NET Core SDK may already be installed on your OS. The .NET Core SDK might have been installed if you installed a development IDE, such as Visual Studio or Visual Studio for Mac. You can check what is installed by opening a terminal and running the `dotnet --list-sdks` command:
+
+::: zone pivot="os-windows"
+
+```console
```

Comment:
Isn't this a dotnetcli ?

Reply:
In the context of this article, it's more appropriate to give the terminal info, since this article has specific outputs based on the terminal/OS being used.